### PR TITLE
Show ert version/komodo version in GUI

### PR DESCRIPTION
**Pull request by yngve-sk**
Show komodo version and hostname if there is a `KOMODO_VERSION` env var
otherwise, show ert version from `ert.shared.__version__`

### CODE REVIEW

**Comment by yngve-sk** [comment]
Screenshot 2024-06-10 at 07 39 07

On RGS with komodoenv

Screenshot 2024-06-10 at 07 56 59

File: src/ert/gui/find_ert_info.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import os
+import socket
+
+from ert.shared import __version__
+
+
+def find_ert_info():
```
Comment (andreas-el) [inline src/ert/gui/find_ert_info.py:7]: `def find_ert_info() -> str:`

**Comment by andreas-el** [comment]
Would it be nicer if removing the parentheses?

**Review by andreas-el — APPROVED** [review]
I support this change. ❤️

**Pull request merged**